### DISCUSSION TITLE
What is ||about:invalid

### DISCUSSION THREAD

**Discussion by peace2000** [uBlockOrigin/uBlock-issues#2275] [Q&A]
Hi, on this page:

https://www.sijoittaja.fi/osakesaastotilisalkut/

When creating a filter by selecting the top bar area in element picker:

![kuva](https://user-images.githubusercontent.com/17256841/191283912-f2e20f36-366d-4412-82e1-d8258e44a6d2.png)

uBO suggests me this filter:

`||about:invalid`

![kuva](https://user-images.githubusercontent.com/17256841/191284492-cc94592c-5245-49bd-a572-932452737a9e.png)

Is this normal behavior?

**Comment by MasterKia** [comment]
Also the page becomes blank when opening uBO's DOM inspector.

**Reply by gorhill** [reply]
uBO extracts the URL from the `background-image` property of the `div` and Firefox says that the property is `url("about:invalid")`.

**Reply by krystian3w** [reply]
So webmaster fail?

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/886325/191682775-26c8e07c-37fb-4b81-bec5-655dea113d2a.png)
![image](https://user-images.githubusercontent.com/886325/191683405-ab4559c2-d2b4-4aa3-bd5c-0654abce1aa0.png)

and after a quick search, I have found this: https://github.com/w3c/csswg-drafts/issues/6447

**Reply by peace2000** [reply]
Thanks for the link.

On Chrome, it suggests a filter that would block the whole url itself:

![image](https://user-images.githubusercontent.com/17256841/191701054-327096ed-024b-45f0-aaab-c428e7af7179.png)

Is there something that should / could be done on uBO to prevent the picker from suggesting these bad filters? Or is this something trivial / wontfix ?

@gorhill

**Reply by MasterKia** [reply]
`https://github.com/uBlockOrigin/uBlock-issues/issues/2278`
